apps/core-app/src/pages/ProposalDetails.tsx

```diff
@@ -51,23 +51,25 @@ export function ProposalDetails() {
   >();
   const [proposalLoading, setProposalLoading] = useState<boolean>(false);
 
+  const fetchProposal = useCallback(() => {
```

Comment:
We are going to have conflicts as i did something similar for my pr to refresh after the cancel proposal tx.

I need to pass a function down to use in the lifecycle. If we can pass this callback down that could replace what I did.

Reply:
i fixed this